test/unit/models/taskRequest.test.js

```diff
@@ -0,0 +1,167 @@
+const chai = require("chai");
+const { expect } = chai;
+
+const cleanDb = require("../../utils/cleanDb");
+const userData = require("../../fixtures/user/user")();
+const addUser = require("../../utils/addUser");
+const taskModel = require("../../../models/tasks");
+const taskData = require("../../fixtures/tasks/tasks")();
+const taskRequestModel = require("../../../models/taskRequests");
+const userModel = require("../../../models/users");
+const { TASK_STATUS } = require("../../../constants/tasks");
+const { TASK_REQUEST_STATUS } = require("../../../constants/taskRequests");
+
+describe("Task Request", function () {
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("addOrUpdate", function () {
+    let user0, user1, task;
+
+    beforeEach(async function () {
+      user0 = await addUser(userData[9]);
+      user1 = await addUser(userData[10]);
+      task = await taskModel.updateTask(taskData[4]);
+    });
+
+    it("should add the task if no taskRequest is present for taskId", async function () {
```

Comment:
should add task or should add request please mention a clear message,

I do not get where should it add

Reply:
addressed, pls check